app/models/bit_core/tool.rb

```diff
@@ -29,6 +29,10 @@ def last_position
       content_modules.order(:position).last.try(:position) || 0
     end
 
+    def slug_title
+      slug || title
```

Comment:
What I mean is: what is the consumer of this method going to do with the result?